build.sbt

```diff
@@ -19,7 +19,10 @@ lazy val core = module("core")
   .dependsOn(meta)
   .settings(libraryDependencies ++= Seq(
     "org.typelevel" %%% "cats-core"   % "1.1.0",
-    "org.typelevel" %%% "cats-free"   % "1.1.0"))
+    "org.typelevel" %%% "cats-free"   % "1.1.0",
+    "org.typelevel" %%% "kittens"     % "1.1.0",
```

Comment:
This brings in the kittens dependency. Long term we should see about removing this dependency and instead having macros to generate Functor/Traverse instances without Shapeless.

For now, can we split fixie into a separate module to better isolate the additional dependency? I need to keep the core module dependency free with the exception of the binary compat guaranteed cats modules.

For splitting, what do you think of `droste-extras` or another similar name?

Reply:
Yeah, makes sense. Maybe `droste-macros` makes more sense? We could put all our macros there and isolate the use of `macro paradise`.